src/utils/network/session-core.js

```diff
@@ -46,15 +45,19 @@ import {
  * @param {Session} session - Solid's Session Object {@link Session}
  * @param {string} fileType - Type of document
  * @param {Access} permissions - The Access object for setting ACL in Solid
- * @param {string} otherPodUsername - Username to other user's Pod or empty string
+ * @param {string} podToSetPermissionsToURL - URL of the other user's Pod to give/revoke permissions OR empty string
  * @returns {Promise} Promise - Sets permission for otherPodUsername for given
  * document type, if exists, or null
  */
-export const setDocAclPermission = async (session, fileType, permissions, otherPodUsername) => {
+export const setDocAclPermission = async (
+  session,
+  fileType,
+  permissions,
+  podToSetPermissionsToURL
+) => {
   const containerUrl = getContainerUrl(session, 'Documents', INTERACTION_TYPES.SELF);
```

Comment:
Think we could also replace the function call for `getContainerUrl` too with `${<podUrl_of_user>}PASS/Documents/` (see example from `setDocContainerAclPermission` in session-core). We just need to pass the user's own podUrl into `setDocAclPermission` as an additional parameter.

Reply:
> Think we could also replace the function call for `getContainerUrl` too with `${<podUrl_of_user>}PASS/Documents/` (see example from `setDocContainerAclPermission` in session-core). We just need to pass the user's own podUrl into `setDocAclPermission` as an additional parameter.

Just for clarity, that should be the podUrl of the logged in user, not the podUrl of the person they want to give/revoke permissions to/from correct?

Reply:
Yeah, the logged in user, which is why it'll be included as an additional parameter. One will be used to build the path for the user's Documents container (`podUrl`), the other will be used to build the other person's webId (`podToSetPermissionsToUrl`).